app/classifier/tasks/generic.cjsx

```diff
@@ -48,6 +48,22 @@ module.exports = React.createClass
             </small>
           </p>
         </div>}
+
+      {if @props.toggleHideMarksBefore
+        <div>
+          <p>
+            <small>
+              <strong>
+                <label className="hide-previous-marks-toggle">
+                  <input type="checkbox" onClick={@props.toggleHideMarksBefore}>
+                    Hide previous marks
+                  </input>
+                </label>
```

Comment:
Hey @saschaishikawa, I would move the <Label /> text out of the `<Input />`, like so:
```
                <label className="hide-previous-marks-toggle">
                  <input type="checkbox" onChange={@props.toggleHideMarksBefore}>
                  </input>
                  {"Hide previous marks"}
                </label>
```
This will get rid of console warning:
```
 `Warning: input is a void element tag and must not have `children``
```